solr/modules/analysis-extras/build.gradle

```diff
@@ -19,6 +19,13 @@ apply plugin: 'java-library'
 
 description = 'Additional analysis components'
 
+configurations.all {
+  resolutionStrategy {
+    force 'org.apache.opennlp:opennlp-tools:2.2.0'
+    force 'org.apache.opennlp:opennlp-dl:2.2.0'
+  }
+}
+
```

Comment:
Why do you need for force a resolution? This shouldn't be necessary at all...

Reply:
We need https://github.com/apache/lucene/pull/448 to be merged to get us to 2.2, or we ahve to work around Lucene...   Is there a better way?   This is what Jeff and I hacked in..

Reply:
try `org.apache.opennlp:opennlp*=2.2.0` in versions.props and remove this from here. It should force the upgrade to 2.2.0.

Reply:
lots of errors when I took out the `resolutionStrategy`:

```
* What went wrong:
Could not determine the dependencies of task ':solr:modules:analysis-extras:compileJava'.
> Could not resolve all task dependencies for configuration ':solr:modules:analysis-extras:compileClasspath'.
   > Could not resolve org.apache.opennlp:opennlp-tools:2.2.0.
     Required by:
         project :solr:modules:analysis-extras
         project :solr:modules:analysis-extras > org.apache.opennlp:opennlp-dl:2.2.0
      > Cannot find a version of 'org.apache.opennlp:opennlp-tools' that satisfies the version constraints:
           Dependency path 'org.apache.solr:analysis-extras:10.0.0-SNAPSHOT' --> 'org.apache.opennlp:opennlp-tools:2.2.0'
           Constraint path 'org.apache.solr:analysis-extras:10.0.0-SNAPSHOT' --> 'org.apache.solr:core:10.0.0-SNAPSHOT' (apiElements) --> 'org.apache.opennlp:opennlp-tools:1.9.4' because of the following reason: Computed from com.palantir.consistent-versions' versions.lock in solr-root
           Constraint path 'org.apache.solr:analysis-extras:10.0.0-SNAPSHOT' --> 'org.apache.solr:solrj:10.0.0-SNAPSHOT' (apiElements) --> 'org.apache.opennlp:opennlp-tools:1.9.4' because of the following reason: Computed from com.palantir.consistent-versions' versions.lock in solr-root
           Dependency path 'org.apache.solr:analysis-extras:10.0.0-SNAPSHOT' --> 'org.apache.lucene:lucene-analysis-opennlp:9.8.0' (compile) --> 'org.apache.opennlp:opennlp-tools:1.9.1'

```

also, feel free to push up changes to this branch ;-)

Reply:
make sure to run `./gradlew --write-locks` to make sure it takes into account `versions.props` changes.